src/Platforms/SqlitePlatform.php

```diff
@@ -328,14 +328,34 @@ protected function _getCommonIntegerTypeDeclarationSQL(array $column)
         return ! empty($column['unsigned']) ? ' UNSIGNED' : '';
     }
 
+    /**
+     * Enables or disables schema emulation (replace dot with 2 underscores).
```

Comment:
"(replace dot with 2 underscores)" is an implementation detail. Please remove.

Reply:
This helps the user to understand the behaviour. It is hardcored as the emulation method is private/final.

If you still belive it is bad, let me know and I will remove.

Reply:
I agree, we should remove "(replace dot with 2 underscores)"